examples/auth/solidstart/.env.example

```
@@ -0,0 +1,3 @@
+# Supabase Configuration
+PUBLIC_SUPABASE_URL=https://slmzeiwqohkvgzembhii.supabase.co
+PUBLIC_SUPABASE_PUBLISHABLE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InNsbXplaXdxb2hrdmd6ZW1iaGlpIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NjcwOTQzMTQsImV4cCI6MjA4MjY3MDMxNH0.bHDFA0JrqhppjQfomsSN6YYSRA9fCxon5MWHNGd1wds
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)